### ISSUE TITLE
Holding Ctrl on a Terraform Resource Name will sometimes open a browser

### ISSUE THREAD

**Issue by walked** [erd0s/terraform-autocomplete#1]
Highlight an AWS resource (e.g. aws_eip), hold control (4-10sec). A browser window will sometimes open as if you had hit F12.

Weird!

I looked at the code and couldnt find an obvious cause but figured I'd let you know!

**Comment by jpSimkins** [comment]
Sometimes? Every time in my case.. I had it open over 20 tabs! Took me a few to identify this was the issue.

**Comment by walked** [comment]
@jpSimkins You're right; if I give it ample time it always happens. It's just frustrating enough to make the extension unusable :(

Maybe I'll run through the source again when I have some downtime.

**Comment by dschaaff** [comment]
I'm experiencing the same thing, but haven't been able to look into the code yet to see if I can identify why.

**Comment by kogent** [comment]
Is there a way to disable just that feature.  I like the autocomplete but I use Dash.app for documentation.

**Comment by jpSimkins** [comment]
I had to disable the plugin. Been waiting for this to be updated before I install it again. I was going to try to work on it myself but I just don't have the time :(

**Comment by erd0s** [comment]
Sorry guys I didn't have my github notifications setup properly so I didn't notice any of this, there's no way to disable it right now without modifying the plugin, the functionality comes from the TerraformDefinitionProvider.ts file, I wanted "go to definition" functionality for the types because I was constantly looking them up on the site, unfortunately this is the only way I could figure out to get some sort of interaction like this. I'll look to making this an option

**Comment by dschaaff** [comment]
This plugin has since added autocomplete and doesn’t have this behavior FYI https://marketplace.visualstudio.com/items?itemName=mauve.terraform
